### PR TITLE
Carousel image ratio

### PR DESCRIPTION
**Pull request by wazolab**

### CODE REVIEW

File: components/PoisDetails/Carousel.vue [hunk 1]
```diff
@@ -63,4 +63,10 @@ export default defineNuxtComponent({
   border: 2px solid #fff;
   border-radius: 100%;
 }
+
+:deep(picture img) {
+  aspect-ratio: 16/9;
+  object-fit: cover;
+  width: 100%;
```
Comment (frodrigo) [inline components/PoisDetails/Carousel.vue:70]: I think only `width` have effect.  `aspect-ratio: 16/9;`. And `object-fit: cover;` it is already inherited from `.tw-object-cover`.

I am not sure if it is an other issue or not. But the download image size if not the right.

Comment (wazolab) [reply components/PoisDetails/Carousel.vue:70]: After checking everything is needed !
- `aspect-ratio` constraint the frame width/height
- `width` tells the frame to take the full widht of parent column.
- `object-fit` tell how the image should cover the frame

**Comment by wazolab** [comment]
I create a separate Issue for `srcset` checks.
I need to read more informations about this feature and how it's supposed to be handled in Nuxt.

**Comment by frodrigo** [comment]
The image does not respect the ratio 16/9.

I have nothing again the current displayed ratio here. But not match the aim of this PR.


http://haute_vienne.localtest.me:3000/poi/29285/details

image


image

**Pull request merged**